README.md

```diff
@@ -162,6 +162,19 @@ Results in the following JSON Schema files:
 
 </details>
 
+## SourceInfo
+
+The sourceinfo plugin provides a protoc-gen-sourceinfo command that generates an `.sourceinfo.binpb` file, for every `.proto` file.
+Each generated file contains the associated serialized `google.protobuf.SourceInfo` value.
```

Comment:
There is no type called `google.protobuf.SourceInfo`. Do you mean `google.protobuf.SourceCodeInfo`? If so, this calls into question the naming of a lot of things here (package, plugin, etc).

Additionally, a `SourceCodeInfo` (if this is the intent) does not make a ton sense without an associated `FileDescriptorProto` - why not make the plugin provide the entire `FileDescriptorProto/FileDescriptorSet`?

Reply:
Nice typo find! I was following @jhump's naming scheme, I could switch it to source _code_ info every where if you think that would provide more clarity.
The normal code gen already comes packed with the Descriptors for normal reflection, this plugin provides just the part that is stripped out. @jhump's sourceinfo library nicely glues everything together at runtime, they same could be done in other languages.